UI/window-basic-preview.hpp

```diff
@@ -58,17 +61,23 @@ class OBSBasicPreview : public OBSQTDisplay {
 	bool locked = false;
 	bool scrollMode = false;
 	bool fixedScaling = false;
+	bool selectionBox = false;
 	int32_t scalingLevel = 0;
 	float scalingAmount = 1.0f;
 
-	obs_sceneitem_t *hoveredPreviewItem = nullptr;
-	obs_sceneitem_t *hoveredListItem = nullptr;
+	DARRAY(obs_sceneitem_t *) hoveredPreviewItems;
+	DARRAY(obs_sceneitem_t *) selectedItems;
```

Comment:
darrays are for C, you should use `std::vector` for dynamic-length arrays in C++.